cdap-ui/app/cdap/components/Tabs/Tabs.scss

```diff
@@ -41,7 +41,7 @@
       width: 100%;
       display: flex;
       &:first-child {
-        height: 35px;
+        height: fit-content;
```

Comment:
Even though the fit-content works only behind vendor flag in firefox I was not able to see anything broken in FF.